### ISSUE TITLE
Laravel 5 Support

### ISSUE THREAD

**Issue by trupedia** [BKWLD/laravel-haml#15]
Hi there,

are you planning to support L5 in a future release?

**Comment by weotch** [comment]
Yes, definitely.  Though I don't expect I'd get to this for a month probably, based on other workload.  I'd happily accept a PR though.

**Comment by weotch** [comment]
@trupedia can you pull from master and verify that everything is still working for you in L5?  I updated your PR so that the package still works for both L4 as well as L5.
